src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -54,6 +54,7 @@ class ResourceType(Enum):  # pylint: disable=too-few-public-methods
     MGMT_APPSERVICE = ('azure.mgmt.web', 'WebSiteManagementClient')
     MGMT_IOTHUB = ('azure.mgmt.iothub', 'IotHubClient')
     MGMT_ARO = ('azure.mgmt.redhatopenshift', 'AzureRedHatOpenShiftClient')
+    MGMT_BATCH = ('azure.mgmt.batch', 'BatchManagementClient')
```

Comment:
Why do you change this file? @Juliehzl

Reply:
Mainly since it is functionally correct, but I'll remove it at your request as this needs to get merged into the current release